common/app/views/fragments/commercial/adSlot.scala.html

```diff
@@ -14,10 +16,15 @@
     class="js-ad-slot ad-slot ad-slot--@name @adTypes.map{ adType =>ad-slot--@adType}.mkString(" ") @optClassNames"
     data-link-name="ad slot @name"
     data-name="@name"
-    @if(!showLabel){ data-label="false" }
-    @if(!refresh){ data-refresh="false" }
-    @if(outOfPage){ data-out-of-page="true" }
-    @sizeMapping.map{ case(breakpoint, sizes) =>
+    @if(!showLabel) { data-label="false" }
+    @if(!refresh) { data-refresh="false" }
+    @if(outOfPage) { data-out-of-page="true" }
+    @sizeMapping.map { case(breakpoint, sizes) =>
         data-@breakpoint="@sizes.mkString("|")"
     }
-    >@placeholderContent</div>
+    >@placeholderContent
+
+    @Commercial.getBlockthroughElementUid(name).map { uid =>
+        <span class="bt-uid-tg" uid="@uid" style="display: none !important"></span>
```

Comment:
You can use `<span hidden ...>` instead of using the `style` attribute

Reply:
This is the tag specified by blockthrough specifically.